packages/js/product-editor/src/components/tabs/tabs.tsx

```diff
@@ -59,6 +59,16 @@ export function Tabs( { onChange = () => {} }: TabsProps ) {
 		child.click();
 	}
 
+	function handleTabClick( tabId: string ) {
+		navigateTo( {
+			url: getNewPath( { tab: tabId } ),
+		} );
+		window.document.documentElement.scrollTo( {
+			top: 0,
+			behavior: 'smooth',
```

Comment:
I don't know if we want this smooth scrolling behavior since this isn't present anywhere else in our UX.

Also `scrollTo` should work fine here, but just noting that the original issue mentioned by @joelclimbsthings in the issue should no longer be a problem as double scrollbars should be fixed.